packages/dapp/src/constants.ts

```diff
@@ -8,7 +8,9 @@ export enum routes {
   REGISTRY_HOME_ROOT = "/registry",
   HOMEPAGE = "/",
   CONTRACT_ADDRESSES = "/contract-addresses",
-  LISTING = "/listing/:listingAddress/:activeTab?",
+  LISTING = "/listing/:listingAddress/:activeTab?/:postId?",
+  LISTING_STORY_BOOST_PAYMENT = "/listing/:listingAddress/:activeTab?/:postId/payment",
+  LISTING_STORY_BOOST_NEWSROOM = "/listing/:listingAddress/:activeTab?/:postId/newsroom",
```

Comment:
Same here - If the newsroom modal is blocked from rendering on listing page you don't need this.